css/css-text-decor/reference/text-decoration-propagation-05-ref.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>CSS Test: text-decoration propagation - Tables</title>
+<link rel="author" title="Timothy Loh" href="mailto:timloh@chromium.org" />
+<link rel="help" href="https://www.w3.org/TR/css-text-decor-3/#line-decoration">
```

Comment:
Typically these are not in the reference file.

Reply:
Hmm.. I copied from -04 test file, should I remove them?

Reply:
No strong opinions here, I typically don't include them, but it's possible they're present in some other reference files

Reply:
Done.